FabricExample/src/ViewFlatteningScreen.tsx

```diff
@@ -58,13 +62,13 @@ function InnerNotFlattenedParentCollapsable() {
 
 function ParentViewNotFlattenedCollapsableFalseDemo() {
   const tap = Gesture.Tap().onStart(() =>
-    console.log(window.performance.now(), 'tap!')
+    console.log(performance.now(), 'tap!')
   );
 
   return (
     <>
       <Text>Parent view not flattened, collapsable=false</Text>
-      <Text>(structure doesn't change)</Text>
+      <Text>(structure doesn&apos;t change)</Text>
```

Comment:
What was wrong with `'`?

Reply:
Don't know and at this point I'm too afraid to ask.